test/e2e/operator_groups_e2e_test.go

```diff
@@ -442,34 +442,38 @@ var _ = Describe("Operator Group", func() {
 			}(ns)
 		}
 
-		// Generate operatorGroupA - OwnNamespace
```

Comment:
If you're going to have concepts in the code like "operatorGroupA" but the value in the cluster will be auto-generated, it's imperative to print that relationship for folks that debug your test.